src/views/activity/activity_item_view.js

```diff
@@ -35,18 +35,15 @@ export default BaseView.extend({
     // and cannot be delegated.
     this.icon = new Image('img');
     this.icon.onload = () => { this.calculateIconColor(); };
-    this.icon.onerror = () => { this.hideErroredFavicon(); };
+    this.icon.onerror = () => {};
```

Comment:
It might be cool to eventually instrument this, but since we're doing nothing with the error handler here, probably best to just get rid of it.